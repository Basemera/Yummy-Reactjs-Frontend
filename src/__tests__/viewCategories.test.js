import React from 'react';
import { shallow, enzyme } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';
import toJson, { shallowToJson } from 'enzyme-to-json';
import sinon from 'sinon';

import Categories from '../components/categories/dashboard';

describe('Signup component', () => {
  const wrapper = shallow(<Categories />);

  it('renders properly', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
