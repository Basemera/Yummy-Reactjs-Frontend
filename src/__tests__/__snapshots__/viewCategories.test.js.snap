// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup component renders properly 1`] = `
<div>
  <CreateCategory
    getCategories={[Function]}
  />
  <div
    className="viewcategories"
  >
    <Search
      handleSearch={[Function]}
      placeholder="categories "
    />
    <div
      className="row categories"
    >
      <div
        className="col-sm-5 offset-sm-6"
      >
        <div
          class="alert alert-info"
          role="alert"
        >
          <strong>
            Ooops!
          </strong>
           No categories , go a head and add some
        </div>
      </div>
    </div>
    <Pagination
      activePage={null}
      changePage={[Function]}
      nextPage={null}
      previousPage={null}
      totalPages={null}
    />
  </div>
</div>
`;
